service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderProgressionService.java

```diff
@@ -27,10 +31,16 @@
     // requires changes in CCD definition. Decided not in scope of 24.
 
     private final ObjectMapper mapper;
+    private final RequestData requestData;
+    private final ApplicationEventPublisher applicationEventPublisher;
 
     @Autowired
-    public CaseManagementOrderProgressionService(ObjectMapper mapper) {
+    public CaseManagementOrderProgressionService(ObjectMapper mapper,
+                                                RequestData requestData,
+                                                ApplicationEventPublisher applicationEventPublisher) {
```

Comment:
I think we might want to consider `@RequiredArgsConstructor(onConstructor = @__(@Autowired))` in place of writing out the `@Autowired` constructor